### PR TITLE
[1.21.2-fabric] Update Japanese translation

### PR DESCRIPTION
**Pull request by momo-i2**
Revert symbols from full-width to half-width
(In Japanese)
楽器名は[こちら](https://minecraft.fandom.com/ja/wiki/%E9%9F%B3%E7%AC%A6%E3%83%96%E3%83%AD%E3%83%83%E3%82%AF#%E6%A5%BD%E5%99%A8)をベースとしているため、混乱を招かないように元に戻した。(custom_headのみ値が長くなるため任意のサウンドとした)

### CODE REVIEW

File: src/main/resources/assets/jade/lang/ja_jp.json [hunk 1]
```diff
@@ -220,27 +271,27 @@
 	"config.jade.plugin_minecraft.energy_storage": "エネルギーストレージ",
 	"config.jade.plugin_minecraft.energy_storage.detailed": "詳細のみ",
 	"config.jade.plugin_minecraft.energy_storage.detailed_desc": "${SHOW_DETAILS}を押した時にのみエネルギー情報を表示します。",
-	"jade.fe": "%s／%s",
-	"config.jade.plugin_minecraft.fluid_storage": "液体ストレージ",
-	"config.jade.plugin_minecraft.fluid_storage_extra_msg": "タンク,液体",
+	"jade.fe": "%s/%s",
+	"config.jade.plugin_minecraft.fluid_storage": "流体ストレージ",
+	"config.jade.plugin_minecraft.fluid_storage_extra_msg": "タンク,流体",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ja_jp.json]: "extra_msg" should be texts that are not contained in the main text, so it should still be "液体" here.

File: src/main/resources/assets/jade/lang/ja_jp.json [hunk 2]
```diff
@@ -253,25 +304,114 @@
...
+	"jade.access.creeper.powered": "帯電%s",
+	"jade.access.zombie_villager.curing": "治療中の%s",
+	"jade.access.goat.hornless": "角無しの%s",
+	"jade.access.wither.powered": "パワード%s",
+	"jade.access.entity.sheared": "毛を刈られた%s",
+	"jade.access.entity.saddled": "鞍付きの%s",
+	"jade.access.entity.baby": "子供の%s",
+	"jade.access.entity.angry": "怒った%s",
+	"jade.access.bee.nectar": "花粉付きの%s",
+	"jade.access.slime.size": "{1,選択,1#小|2≤中|4≤大|8≤巨大} {0}",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ja_jp.json]: "choice" should not be translated

File: src/main/resources/assets/jade/lang/ja_jp.json [hunk 3]
```diff
@@ -253,25 +304,114 @@
...
-	"__comment": "Only for testing:",
+	"jade.collectingItems": "アイテムを収集中...",
+	"config.jade.plugin_jade.block_properties": "ブロックプロパティ",
+	"jade.block_explosion_resistance": "爆発耐性: %s",
+	"jade.block_destroy_time": "破壊時間: %s",
+	"jade.block_jump_factor": "跳躍係数: %s",
+	"jade.block_speed_factor": "速度係数: %s",
+	"jade.block_ignite_odds": "発火確率: %s",
+	"jade.block_burn_odds": "燃焼確率: %s",
+	"jade.ignore_list.comment": "これはJadeのターゲットの無視リストです。レジストリIDを「値」リストに追加できます。",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ja_jp.json]: "values" should not be translated

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/ja_jp.json [hunk 4]
```diff
@@ -139,57 +186,61 @@
...
+	"jade.instrument.dragon": "エンダードラゴンの咆哮",
+	"jade.instrument.wither_skeleton": "ウィザースケルトンの骨を鳴らす音",
+	"jade.instrument.piglin": "ピグリンの鼻を鳴らす音",
+	"jade.instrument.custom_head": "任意のサウンド",
 	"config.jade.plugin_minecraft.armor_stand": "防具立て",
 	"config.jade.plugin_minecraft.painting": "絵画",
-	"jade.nextEgg": "産卵まで：%s",
+	"config.jade.plugin_minecraft.next_entity_drop": "次のエンティティドロップ",
+	"config.jade.plugin_minecraft.next_entity_drop_desc": "卵やアルマジロのウロコなどの次のエンティティがドロップされるまでの時間を表示します。",
+	"config.jade.plugin_minecraft.next_entity_drop_extra_msg": "egg,scute",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ja_jp.json]: seems untranslated due to an oversight?

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/ja_jp.json (hunk 1 above)
Comment (momo-i2) [reply src/main/resources/assets/jade/lang/ja_jp.json]: Fixed!

File: src/main/resources/assets/jade/lang/ja_jp.json (hunk 2 above)
Comment (momo-i2) [reply src/main/resources/assets/jade/lang/ja_jp.json]: Fixed!

File: src/main/resources/assets/jade/lang/ja_jp.json (hunk 3 above)
Comment (momo-i2) [reply src/main/resources/assets/jade/lang/ja_jp.json]: Fixed!

File: src/main/resources/assets/jade/lang/ja_jp.json (hunk 4 above)
Comment (momo-i2) [reply src/main/resources/assets/jade/lang/ja_jp.json]: Fixed!

**Comment by momo-i2** [comment]
Is the key “jade.waxed” only used for Sign? If so, the translation needs to be corrected.

**Comment by Snownee** [comment]
> Is the key “jade.waxed” only used for Sign? If so, the translation needs to be corrected.

In Jade itself, yes. But the translation can be used by other mods

**Comment by momo-i2** [comment]
> > Is the key “jade.waxed” only used for Sign? If so, the translation needs to be corrected.
> 
> In Jade itself, yes. But the translation can be used by other mods
Thank you for answer, I will fix that. It will be last commit in next.

**Comment by momo-i2** [comment]
> Thank you for answer, I will fix that. It will be last commit in next.

Sorry, I'll take back my previous comment (the last commit), and I hope you will merge it with the 1.21.4 neoforge version when it is released.
Furthermore, until it is released, I will investigate as much as possible about the strings that are unknown in the game (because I left them to machine translation), and if there is any difficulty, please let me confirm it on the Discord.

**Pull request merged**